exasol/nb_connector/cli/processing/processing.py

```diff
@@ -60,20 +64,55 @@ def save(
             continue
         if secret := option.get_secret(interactive=bool(value)):
             scs.save(option.scs_key, secret)
-    return 0
 
 
-def show_scs_content(scs_file: Path) -> int:
+def verify_connection(scs: Secrets) -> None:
+    """
+    Verify if successful connection to the configured backend is possible.
+    Raise an ScsCliError otherwise.
+    """
+    if BackendSelector(scs).use_itde:
+        # Question: Is it OK, to let bring_itde_up modify the SCS content, here?
+        False and bring_itde_up(scs)
+        report.warning(f"Bring up ITDE currently disabled")
+        return
+    try:
+        open_pyexasol_connection(scs).execute("SELECT 1 FROM DUAL").fetchone()
```

Comment:
Should `verify_connection()` (CLI command `scs check --connect`) verify the bucketfs, too?

Reply:
Yes

Reply:
- Is it sufficient to list the BFS contents in the test? 
- What is the expectation? 
- Are there any default files we expect to be contained in the BFS? 
- Or is it acceptable to modify the BFS contents for the sake of testing the connection?  
- Should the test also do a cleanup after having verified the connection?

Reply:
I moved the BucketFS verification to a separate ticket/PR:
* #274